src/webgpu/api/validation/resource_usages/texture/in_render_common.spec.ts

```diff
@@ -95,3 +101,128 @@ g.test('subresources_from_same_texture_as_color_attachments')
       encoder.finish();
     }, !success);
   });
+
+// Test that when one subresource of a texture is used as a color attachment, it cannot be used in a
+// bind group simultaneously in the same render pass encoder. It is allowed when the bind group is
+// used in another render pass encoder instead of the same one.
+g.test('subresources_from_same_texture_as_color_attachment_and_in_bind_group')
+  .params(u =>
+    u
+      .combine('colorAttachmentBaseLevel', [0, 1])
+      .combine('colorAttachmentBaseLayer', [0, 1])
+      .combine('bindGroupViewBaseLevel', [0, 1, 2])
+      .combine('bindGroupViewLevelCount', [1, 2])
+      .combine('bindGroupViewBaseLayer', [0, 1, 2])
+      .combine('bindGroupViewLayerCount', [1, 2])
```

Comment:
nit: The full combination of all of these parameters is probably not useful; perhaps consider something like
```
.combineWithParams([
  { bindGroupViewBaseLevel: 0, bindGroupViewLevelCount: 1 },
  { bindGroupViewBaseLevel: 1, bindGroupViewLevelCount: 1 },
  { bindGroupViewBaseLevel: 1, bindGroupViewLevelCount: 2 },
]).combineWithParams([
  { bindGroupViewBaseLayer: 0, bindGroupViewLayerCount: 1 },
  { bindGroupViewBaseLayer: 1, bindGroupViewLayerCount: 1 },
  { bindGroupViewBaseLayer: 1, bindGroupViewLayerCount: 2 },
])
```
Then you also can simplify the `.unless`.